test/models/post_test.rb

```diff
@@ -18,4 +18,13 @@ def setup
       @Post.content = "    "
       assert_not @Post.valid?
   end
+
+  test "post can have comments" do
+    comment = comments(:commentjs)
```

Comment:
Nice job. One style note is that variables and methods should always be lowercase. ClassNames are capitalized, and CONSTANTS are all caps.